<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QMainWindow" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>621</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>5</number>
       </property>
       <widget class="QWidget" name="main">
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>50</y>
           <width>361</width>
           <height>361</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="host_btn">
            <property name="text">
             <string>Host a game</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="join_btn">
            <property name="text">
             <string>Join a game</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="host_page">
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>49</x>
           <y>19</y>
           <width>391</width>
           <height>371</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>player numbers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="number">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>7</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="host">
            <property name="text">
             <string>host this game</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="host_cancel">
            <property name="text">
             <string>cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="join_wait">
        <widget class="QWidget" name="verticalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>441</width>
           <height>361</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QListWidget" name="players"/>
          </item>
          <item>
           <widget class="QPushButton" name="cancel_join">
            <property name="text">
             <string>cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="host_wait">
        <widget class="QWidget" name="verticalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>29</x>
           <y>19</y>
           <width>411</width>
           <height>351</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QListWidget" name="invited_players"/>
          </item>
          <item>
           <widget class="QPushButton" name="start_game">
            <property name="text">
             <string>start this game</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancel_host">
            <property name="text">
             <string>cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="join_page">
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>69</x>
           <y>60</y>
           <width>331</width>
           <height>331</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QListWidget" name="hosts"/>
          </item>
          <item>
           <widget class="QPushButton" name="join">
            <property name="text">
             <string>Join the select game</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="join_cancel">
            <property name="text">
             <string>cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="play_page">
        <widget class="QWidget" name="verticalLayoutWidget_7">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>10</y>
           <width>501</width>
           <height>471</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="ismyturn">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="points"/>
          </item>
          <item>
           <widget class="QLabel" name="pattern">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="hand"/>
          </item>
          <item>
           <widget class="QPushButton" name="play">
            <property name="text">
             <string>play select card</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uncover">
            <property name="text">
             <string>uncover a card</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
